cfme/utils/template/ec2.py

```diff
@@ -49,25 +49,47 @@ def upload_image(self):
 
     @log_wrap("import image from bucket")
     def import_image(self):
-        try:
-            import_task_id = self.mgmt.import_image(s3bucket=self.bucket_name,
-                                                    s3key=self.template_name,
-                                                    description=self.template_name)
-
-            wait_for(self.mgmt.get_image_id_if_import_completed,
-                     func_args=[import_task_id],
-                     fail_condition=False,
-                     delay=5,
-                     timeout='90m',
-                     message='Importing image to EC2')
-
-            ami_id = self.mgmt.get_image_id_if_import_completed(import_task_id)
-            ami = EC2Image(uuid=ami_id, system=self.mgmt)
-            ami.set_tag("Name", self.template_name)
-            return True
+        if self.stream == 'downstream-510z':
```

Comment:
Maybe create another method for 5.11+ and this condition would be in run method?

Reply:
I think its fine where it is, the logic is specific to the import step.